src/redux/books/books.js

```diff
@@ -1,14 +1,29 @@
-import { ADD_BOOK, REMOVE_BOOK } from '../constants';
+import { ADD_BOOK, ALL_BOOKS, REMOVE_BOOK } from '../constants';
+import { postBooks, deleteBook, getBooks } from '../../components/connectAPI';
 
-export const AddBook = (data) => ({
-  type: ADD_BOOK,
-  data,
-});
+export const AddBook = (data) => async (dispatch) => {
```

Comment:
- [ ] As required, I kindly suggest that you make use of createAsyncThunk for thunk action creators